pkg/network/render.go

```diff
@@ -385,6 +388,61 @@ func isNetworkChangeSafe(prev, next *operv1.NetworkSpec, infraRes *bootstrap.Inf
 	return nil
 }
 
+func isClusterNetworkChangeSafe(prev, next *operv1.NetworkSpec) error {
+
+	// quick check to make sure clusterNetwork slices are of same size as we do not
+	// support adding/removing additional clusterNetwork entries unless it's for a
+	// single/dual stack migration. in those cases validation is done in isNetworkChangeSafe()
+	if len(prev.ClusterNetwork) != len(next.ClusterNetwork) {
+		return errors.Errorf("Adding/Removing clusterNetwork entries of the same type is not supported")
```

Comment:
I think in general errors should not be capitalized: https://github.com/golang/go/wiki/CodeReviewComments#error-strings